git_services/git_services/sidecar/run_command.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+from git_services.git_services.sidecar.config import config_from_env
```

Comment:
```suggestion
from git_services.sidecar.config import config_from_env
```

This is def a problem. But I saw this only after changing the `terminationGracePeriodSeconds` on the session pod. This causes everything to "hang" and I got a shell into the sidecar and tried to run the autosave command manually. This is actually how you can test things out without shutting down the session. I will see if this is the only problem.

Getting caught up on small things like this which are obvious only when you deploy something is unfortunately common in the notebook service IMO.

Reply:
ah I already fixed that in the rpcserver.py but must have missed it here